src/main/java/seedu/address/logic/commands/MarkPaidCommand.java

```diff
@@ -7,6 +7,7 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
+import java.util.regex.Pattern;
```

Comment:
Regex should not be used in *Command.java files. Convention for this project is to use it only in *CommadParser